.taskcluster.yml

```diff
@@ -123,99 +123,126 @@ tasks:
-          then:
-            $mergeDeep:
-              - {$eval: 'default_task_definition'}
-              - scopes:
-                - ${assume_scope_prefix}:release
-                payload:
-                  command:
-                    - >-
-                      git fetch ${repository} refs/tags/${head_rev}
-                      && git config advice.detachedHead false
-                      && git checkout FETCH_HEAD
-                      && python automation/taskcluster/decision_task.py beta ${event.release.tag_name}
-                extra:
-                  treeherder:
-                    symbol: beta-D
-                metadata:
-                  name: Fenix Beta Decision Task
-                  description: Building and releasing Fenix to the beta channel - triggered by release ${event.release.tag_name}
-        - $if: 'tasks_for == "cron"'
-          then:
-            $mergeDeep:
-              - {$eval: 'default_task_definition'}
-              - scopes:
-                  - $if: 'trust_level == 3'
-                    then: assume:hook-id:project-mobile/fenix-nightly
-                    else: assume:hook-id:project-mobile/fenix-nightly-staging
-                routes:
-                  $if: 'trust_level == 3'
-                  then:
-                    - notify.email.fenix-eng-notifications@mozilla.com.on-failed
-                payload:
-                  $let:
-                    staging_flag:
-                      $if: 'trust_level == 3'
-                      then: ''
-                      else: '--staging'
-                  in:
+                    name: Fenix - Decision task
+                    description: Schedules the build and test tasks for Fenix.
+          - $if: 'tasks_for == "github-release" && event["action"] == "published"'
+            then:
+              $mergeDeep:
+                - {$eval: 'default_task_definition'}
+                - scopes:
+                  - ${assume_scope_prefix}:release
+                  payload:
                     command:
                       - >-
-                        git fetch ${repository} ${head_branch}
+                        git fetch ${repository} refs/tags/${head_rev}
                         && git config advice.detachedHead false
                         && git checkout FETCH_HEAD
-                        && python automation/taskcluster/decision_task.py nightly ${staging_flag}
-                extra:
-                  cron: {$json: {$eval: 'cron'}}
-                  treeherder:
-                    symbol: nightly-D
-                metadata:
-                  name: Fenix Nightly Decision Task
-                  description: Decision task scheduled by cron task [${cron.task_id}](https://tools.taskcluster.net/tasks/${cron.task_id})
+                        && python automation/taskcluster/decision_task.py beta ${event.release.tag_name}
+                  extra:
+                    treeherder:
+                      symbol: beta-D
```

Comment:
Just to clarify: good for now, but at some point we'll have to rewrite these once we'll need to ship major releases as well, not just betas, right?

Reply:
Yeah, I'll probably have to call this something non-track-specific, like `D`.

Reply:
SGTM!